akka-http/src/main/scala/akka/http/scaladsl/server/ContentNegotation.scala

```diff
@@ -21,10 +21,10 @@ final class MediaTypeNegotiator(requestHeaders: Seq[HttpHeader]) {
     (for {
       Accept(mediaRanges) ← requestHeaders
       range ← mediaRanges
-    } yield range).sortBy { // `sortBy` is stable, i.e. upholds the original order on identical keys
-      case x if x.isWildcard     ⇒ 2f // most general, needs to come last
-      case MediaRange.One(_, qv) ⇒ -qv // most specific, needs to come first
-      case _                     ⇒ 1f // simple range like `image/*`
+    } yield range).sortBy { // `sortBy` is stable, i.e. upholds the original order on identical weights
+      case x if x.isWildcard   ⇒ 2f // most general, needs to come last
```

Comment:
What if that has params as well? Should that go into the specificity calculation?